nsc.hcl

```diff
@@ -0,0 +1,18 @@
+description = "a client for Namespace's cloud"
+homepage = "https://github.com/namespacelabs/foundation"
+binaries = ["nsc"]
+test = "nsc -h"
+
+platform "amd64" {
```

Comment:
Better to remove these platform lines so we get arm64 on Mac/Linux

Reply:
Just a top-level `source` block?